pagopa/io-app
# fix(IT Wallet): [SIW-1715] TS credential barcode format

## Short description
This PR updates the format encoding for the TS credential barcode which is supposed to be `CODE39` according to the [documentation](https://sistemats1.sanita.finanze.it/portale/documents/20182/34254/allegato%2Btecnico%2BTS-CNS%2Bex%2BDL%2B78-2010_v22-06-12.pdf/2ef2b969-879c-64f5-2b0a-8bce9877c08f). The library used defaults to `CODE128` instead. This causes an issue with scanner which do not support `CODE128` encoding.

## List of changes proposed in this pull request
- Specify `CODE39` as barcode format;
- Decrease the line width accordingly.

## How to test
- Open the app;
- Activate the wallet;
- Obtain a TS credential;
- Open the credential details and check if the barcode is rendered correctly.
